src/constants/common.js

```diff
@@ -123,6 +123,21 @@ module.exports = {
 	INVITED: 'INVITED',
 	ENROLLED: 'ENROLLED',
 	UNIT_OF_TIME: 'minutes',
+	fileTypeCSV: 'text/csv',
+	NO_OF_ATTEMPTS: 3,
+	BACK_OFF_RETRY_QUEUE: 600000,
+	tempFolderForBulkUpload: 'public/invites',
+	sessionOutputFile: 'output-session-creation',
+	csvExtension: '.csv',
+	responseType: 'stream',
+	azureBlobType: 'BlockBlob',
+	UPLOADED_STATUS: 'UPLOADED',
+	FAILED_STATUS: 'FAILED',
+	PROCESSED_STATUS: 'PROCESSED',
+	notificationEmailType: 'email',
+	STATUSCODE: 201,
+	CREATEDRESPONSECODE: 'OK',
```

Comment:
Why do we have these response codes etc in common.js? Instead of generics/http-status.js

Reply:
@nevil-mathew will use it from generics file .